ad_rss/src/situation/RssStructuredSceneIntersectionChecker.cpp

```diff
@@ -155,8 +158,10 @@ bool RssStructuredSceneIntersectionChecker::checkIntersectionSafe(Situation cons
                                                           isSafe);
 
       data_intersection.longitudinal_relative_position = "EgoInFront";
```

Comment:
Is it possible to use `to_string()` statement?

Reply:
Could be used however the  `InFront` / `OverlapBack` ... without the "Ego" added is more informative to me